pyuploadcare/api_resources.py

```diff
@@ -231,24 +231,76 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
-        """Creates File copy
+    def local_copy(self, effects=None, store=None):
+        """Creates a Local File Copy on Uploadcare Storage
 
-        If ``target`` is ``None``, copy file to Uploadcare storage otherwise
-        copy to target associated with project.
-        Add ``effects`` to ``self.default_effects`` if any.
+        - effects:
+            Adds CDN image effects to ``self.default_effects`` if any;
+        - store:
+            If ``store`` option is set to False the copy of your file will be deleted
+            in 24 hour period after the upload.
         """
         effects = effects or ''
+        store = store or ''
         if self.default_effects is not None:
             fmt = '{head}-/{tail}' if effects else '{head}'
             effects = fmt.format(head=self.default_effects,
                                  tail=effects)
         data = {
             'source': self.cdn_path(effects)
         }
+        if store is not None:
+            data['store'] = store
+        return rest_request('POST', 'files/', data=data)
+
+    def remote_copy(self, target, effects=None, make_public=None, pattern=None):
+        """Creates File Copy On Remote Storage
+
+        - effects:
+            Adds CDN image effects to ``self.default_effects`` if any;
+        - target:
+            If ``target`` is ``None``, copy file to Uploadcare storage otherwise
+            copy to target associated with project;
+        - make_public:
+            To forbid a public access for your files on the target storage set
+            ``make_public`` option to be False, relevant when ``target`` has a value set.
+            Default value is True.
+        - pattern:
+            Specify  ``pattern`` option to set S3 object key name. Takes precedence over
+            pattern set in the settings. If neither is specified defaults to ${uuid}/${auto_filename}.
+            Relevant to custom storage only.
+
+        For more information on each of the options above please refer to
+        our docs https://uploadcare.com/documentation/rest/#file.
+
+        Following example copies a file to S3 bucket named ``samplefs`` with  owner only access,
+        and file name consisting of UUID concatenated to original filename. The file is placed
+        in the root of the storage:
+             >>>file.remote_copy(target='samplefs',
+                                make_public=False,
+                                pattern='${uuid}${filename}${ext}');
```

Comment:
wanted to demonstrate the filename can be set to a different value from the default one